sql-plugin/src/main/scala/com/nvidia/spark/rapids/basicPhysicalOperators.scala

```diff
@@ -118,6 +118,32 @@ object GpuProjectExec extends Arm {
   }
```

Comment:
2023 copyrights

Reply:
nice catch, updated.
Seems the `auto-copyright` does not work well on IOS.